lib/ui/creators_corner_page.dart

```diff
@@ -1,16 +1,43 @@
 import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/widgets/creators_corner/exercise_library.dart';
+import 'package:workoutpersonalizer_frontend/widgets/creators_corner/workout_editor.dart';
+import 'package:workoutpersonalizer_frontend/widgets/creators_corner/workout_viewer.dart';
 
 class CreatorsCornerPage extends StatelessWidget {
+  const CreatorsCornerPage({Key? key}) : super(key: key);
+
   @override
   Widget build(BuildContext context) {
-    // TODO: WP-13 (ubclaunchpad.atlassian.net/browse/WP-13)
-    return Center(
-      child: Container(
-        child: const Text(
-          "Creator's Corner Page",
-          style: TextStyle(fontSize: 30),
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: <Widget>[
+        Expanded(
+          flex: 6,
```

Comment:
Small code smell but probably better to define these numbers as constants at the top